fixtures/dynamicTemplates.js

```diff
@@ -27,6 +16,20 @@ const dynamic_templates = [
       mapping: { type: 'text' }
     }
   },
+
+  // Projection Extension https://github.com/stac-extensions/projection
+  {
+    proj_epsg: {
+      match: 'proj:epsg',
+      mapping: { type: 'integer' }
+    }
+  },
+  {
+    proj_projjson: {
```

Comment:
this is a complex object with several different variations. attempting to index a variation that is different than the first one indexed can result in an error, see #108